.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = tab
+indent_size = 2
```

Comment:
I've never used editorconfig before. Does this make my tabs 2  units wide or 4 units wide?

I'm assuming 2, but it's better to ask.

Reply:
This just makes it *look like* tabs are 2 spaces wide, but doesn't actually add extra characters.

Reply:
Ah, alright, that's fine then.